src/dayview.html

```diff
@@ -1,65 +1,167 @@
+                    </ng-template>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+        <init-position-scroll *ngIf="sliderIndex===currentViewIndex" class="dayview-normal-event-container"
+          [initPosition]="initScrollPosition" [emitEvent]="preserveScrollPosition"
+          (onScroll)="setScrollPosition($event)">
+          <table class="table table-bordered table-fixed dayview-normal-event-table">
+            <tbody>
+              <tr *ngFor="let tm of views[sliderIndex].rows; let i = index">
+                <td class="calendar-hour-column text-center">
+                  {{hourColumnLabels[i]}}
+                </td>
+                <td class="calendar-cell" tappable (click)="select(tm)">
+                  <ng-template [ngTemplateOutlet]="dayviewNormalEventSectionTemplate"
+                    [ngTemplateOutletContext]="{tm:tm, hourParts: hourParts, eventTemplate:dayviewNormalEventTemplate}">
+                  </ng-template>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </init-position-scroll>
+        <init-position-scroll *ngIf="sliderIndex!==currentViewIndex" class="dayview-normal-event-container"
+          [initPosition]="initScrollPosition">
+          <table class="table table-bordered table-fixed dayview-normal-event-table">
+            <tbody>
+              <tr *ngFor="let tm of views[sliderIndex].rows; let i = index">
+                <td class="calendar-hour-column text-center">
+                  {{hourColumnLabels[i]}}
+                </td>
+                <td class="calendar-cell">
+                  <ng-template [ngTemplateOutlet]="dayviewInactiveNormalEventSectionTemplate"
+                    [ngTemplateOutletContext]="{tm:tm, hourParts: hourParts}">
+                  </ng-template>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </init-position-scroll>
+      </ng-container>
+
+      <ng-container *ngIf="dayviewShowCategoryView">
+        <div *ngIf="sliderIndex===currentViewIndex" class="dayview-category-header">
+          <table class="table table-fixed dayview-category-header-table">
+            <thead>
+              <tr>
+                <th class="dayview-category-header-item calendar-hour-column"></th>
+                <th *ngFor="let category of views[sliderIndex].categories" class="dayview-category-header-item">
+                  <div class="dayview-category-name">
+                    <ng-template [ngTemplateOutlet]="dayviewCategoryItemTemplate"
+                      [ngTemplateOutletContext]="{category}">
+                    </ng-template>
+                  </div>
+                </th>
+              </tr>
+            </thead>
+          </table>
+        </div>
+        <div *ngIf="sliderIndex!==currentViewIndex" class="dayview-inactive-category-header"></div>
```

Comment:
for the inactive view, need to have the same layout as active view.

Reply:
resolved